testing/helpers.ts

```diff
@@ -31,6 +34,8 @@ type JHipsterRunResult<GeneratorType extends CoreGenerator = CoreGenerator> = Ru
    * Composed generators that were mocked.
    */
   composedMockedGenerators: string[];
+
+  askedQuestions: Question[];
```

Comment:
Not needed:
https://github.com/yeoman/yeoman-test/blob/60a2521773294a26402d15c32751765be4f7ca0f/src/run-result.ts#L65